app/presenters/hyrax/admin/users_presenter.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+module Hyrax
+  module Admin
+    class UsersPresenter
+      # @return [Array] an array of Users & and array of roles
+      def users
+        @users ||= search
+      end
+
+      def roles
```

Comment:
Are we going to need a way to reset this if we add a new role?

Reply:
i.e. if you look at a user, add a role, add the user to that role, and come back to the user - will the original role list still be memoized without the new role?

Reply:
yeah, that would be an issue, but this is c&p from hyrax, we could probably fix that.